cob_base_drive_chain/ros/src/cob_base_drive_chain.cpp

```diff
@@ -515,11 +515,10 @@ class NodeClass
 			}
 			else
 			{
-				ROS_WARN("...base already recovered...");
-				res.success.data = true;
-				res.error_message.data = "base already recovered";
+				ROS_WARN("Base not yet initialized.");
+				res.success.data = false;
```

Comment:
why did you change the success behaviour? 
The ```res.success.data``` should contain the state of the base which means ```true``` if the base can be moved afterwards (recover successfully) and ```false``` if the base cannot be moved (recover failed).